<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Preview Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>250</red>
         <green>250</green>
         <blue>250</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>88</red>
         <green>88</green>
         <blue>88</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>144</red>
         <green>144</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>250</red>
         <green>250</green>
         <blue>250</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>45</red>
         <green>45</green>
         <blue>45</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>250</red>
         <green>250</green>
         <blue>250</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>88</red>
         <green>88</green>
         <blue>88</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>144</red>
         <green>144</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>250</red>
         <green>250</green>
         <blue>250</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>45</red>
         <green>45</green>
         <blue>45</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>88</red>
         <green>88</green>
         <blue>88</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>144</red>
         <green>144</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>45</red>
         <green>45</green>
         <blue>45</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>45</red>
         <green>45</green>
         <blue>45</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="GraphicsView" name="graphicsView">
      <property name="toolTip">
       <string/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_infobar" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_help">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>14</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Help</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {

padding: 2px;

color: #fff;
background-color: #2d2d2d;

/*
background: qradialgradient(cx: 0.5, cy: -0.3,
fx: 0.5, fy: -0.3,
radius: 1.4, stop: 0 #888, stop: 1 #222);
*/

border: 1px solid;
border-color: #555;
border-radius: 3px;

height: 16px;
}

QPushButton::hover {

background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #a6a6a6, stop: 0.08 #7f7f7f,
stop: 0.39999 #68717b, stop: 0.4 #4a5e75,
stop: 0.9 #4c4c4c, stop: 1 #333333);

/*
background: qradialgradient(cx: 0.5, cy: -0.3,
fx: 0.5, fy: -0.3,
radius: 1.4, stop: 0 #5a84ae, stop: 1 #222);
*/

}</string>
         </property>
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_size">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>File dimensions</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>0 x 0</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_fileSize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Filesize</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>0 kB</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_scale">
         <property name="toolTip">
          <string>Zoom level</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QDoubleSpinBox {
	background-color: #222;
	color: #eee;
	border
}

QDoubleSpinBox::up-arrow {
 }

QDoubleSpinBox::down-arrow {
 }</string>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="specialValueText">
          <string/>
         </property>
         <property name="accelerated">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>9999.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>10.000000000000000</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_resetZoom">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Reset zoom to the original image size</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {

padding: 2px;

color: #fff;
background-color: rgb(88, 88, 88);

/*
background: qradialgradient(cx: 0.5, cy: -0.3,
fx: 0.5, fy: -0.3,
radius: 1.4, stop: 0 #888, stop: 1 #222);
*/

border: 1px solid;
border-color: #777;
border-radius: 3px;

height: 16px;
}

QPushButton::hover {

background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #a6a6a6, stop: 0.08 #7f7f7f,
stop: 0.39999 #68717b, stop: 0.4 #4a5e75,
stop: 0.9 #4c4c4c, stop: 1 #333333);

/*
background: qradialgradient(cx: 0.5, cy: -0.3,
fx: 0.5, fy: -0.3,
radius: 1.4, stop: 0 #5a84ae, stop: 1 #222);
*/

}</string>
         </property>
         <property name="text">
          <string>1:1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_fitIntoView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Fit image into the preview window</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {

padding: 2px;

color: #fff;
background-color: rgb(88, 88, 88);

/*
background: qradialgradient(cx: 0.5, cy: -0.3,
fx: 0.5, fy: -0.3,
radius: 1.4, stop: 0 #888, stop: 1 #222);
*/

border: 1px solid;
border-color: #777;
border-radius: 3px;

height: 16px;
}

QPushButton::hover {

background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #a6a6a6, stop: 0.08 #7f7f7f,
stop: 0.39999 #68717b, stop: 0.4 #4a5e75,
stop: 0.9 #4c4c4c, stop: 1 #333333);

/*
background: qradialgradient(cx: 0.5, cy: -0.3,
fx: 0.5, fy: -0.3,
radius: 1.4, stop: 0 #5a84ae, stop: 1 #222);
*/

}</string>
         </property>
         <property name="text">
          <string>[  ]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_marked">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>581</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_drag">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Click and drag to a file browser</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {

padding: 2px;

color: #fff;
background-color: rgb(88, 88, 88);

/*
background: qradialgradient(cx: 0.5, cy: -0.3,
fx: 0.5, fy: -0.3,
radius: 1.4, stop: 0 #888, stop: 1 #222);
*/

border: 1px solid;
border-color: #777;
border-radius: 3px;

height: 16px;
}

QPushButton::hover {

background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #a6a6a6, stop: 0.08 #7f7f7f,
stop: 0.39999 #68717b, stop: 0.4 #4a5e75,
stop: 0.9 #4c4c4c, stop: 1 #333333);

/*
background: qradialgradient(cx: 0.5, cy: -0.3,
fx: 0.5, fy: -0.3,
radius: 1.4, stop: 0 #5a84ae, stop: 1 #222);
*/

}</string>
         </property>
         <property name="text">
          <string>Drag to Folder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_openFolder">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Opens the folder containing the current image</string>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {

padding: 2px;

color: #fff;
	background-color: rgb(88, 88, 88);

/*
background: qradialgradient(cx: 0.5, cy: -0.3,
fx: 0.5, fy: -0.3,
radius: 1.4, stop: 0 #888, stop: 1 #222);
*/

border: 1px solid;
border-color: #777;
border-radius: 3px;

height: 16px;
}

QPushButton::hover {

background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #a6a6a6, stop: 0.08 #7f7f7f,
stop: 0.39999 #68717b, stop: 0.4 #4a5e75,
stop: 0.9 #4c4c4c, stop: 1 #333333);

/*
background: qradialgradient(cx: 0.5, cy: -0.3,
fx: 0.5, fy: -0.3,
radius: 1.4, stop: 0 #5a84ae, stop: 1 #222);
*/

}</string>
         </property>
         <property name="text">
          <string>Open Folder</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionKeyboard_Shortcuts">
   <property name="text">
    <string>Keyboard Shortcuts</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen_Image">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
